charts/folder-config-connector/values.yaml

```diff
@@ -0,0 +1,61 @@
+# -- detailed description of the parameters: https://cloud.google.com/config-connector/docs/reference/resource-docs/resourcemanager/folder
+
+# -- Single folder name. If set, "folders" key has to be unset. 
```

Comment:
ja jakoś nie czuję tego podziału na folder i folders. Wg mnie powinna byc tylko opcja folders, ale to moje subiektyne zdanie.

Reply:
It can exists in parallel. if it is not used, it will be deleted in the future